lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivation/IcarusSpec.hs

```diff
@@ -69,24 +70,10 @@ import qualified Data.Text as T
 spec :: Spec
 spec = do
     describe "Golden Tests - Icarus' style addresses" $ do
-        let seed0 = Passphrase "4\175\242L\184\243\191 \169]\171 \207\r\v\233\NUL~&\ETB"
-
-        goldenAddressGeneration $ GoldenAddressGeneration
-            seed0 (toEnum 0x80000000) UTxOExternal (toEnum 0x00000000)
-            "Ae2tdPwUPEZGQVrA6qKreDzdtYxcWMMrpTFYCpFcuJfhJBEfoeiuW4MtaXZ"
-
-        goldenAddressGeneration $ GoldenAddressGeneration
-            seed0 (toEnum 0x80000000) UTxOExternal (toEnum 0x0000000E)
-            "Ae2tdPwUPEZDLWQQEBR1UW7HeXJVaqUnuw8DUFu52TDWCJbxbkCyQYyxckP"
-
-        goldenAddressGeneration $ GoldenAddressGeneration
-            seed0 (toEnum 0x8000000E) UTxOInternal (toEnum 0x0000002A)
-            "Ae2tdPwUPEZFRbyhz3cpfC2CumGzNkFBN2L42rcUc2yjQpEkxDbkPodpMAi"
```

Comment:
Hmmmm... Why ? We can't just get rid of golden tests like this :|

Reply:
Re-added. Originally I didn't see how we'd go from entropy to `SomeMnemonic`.

Reply:
Yeah, the mnemonic is basically the entropy + checksum encoded as words. The checksum can be computed from the entropy itself, so entropy and mnemonic are somewhat equivalent,